### DISCUSSION TITLE
Types of Ml

### DISCUSSION THREAD

**Discussion by mssanamateen** [mssanamateen/Artificial-Intelligence-Lab#43] [Q&A] [answered]
What is unsupervised learning?

**Comment by 24-Vishnu** [comment, marked as answer]
Unsupervised learning is a type of machine learning where the algorithm learns from data without being given specific instructions or labels. Instead of being told what to look for, the model finds patterns or groupings in the data on its own. For example, it might group similar items together or identify trends, like clustering customers based on their buying habits.

**Comment by 24-Vishnu** [comment]
Unsupervised learning is a type of machine learning where the algorithm learns from data without being given specific instructions or labels. Instead of being told what to look for, the model finds patterns or groupings in the data on its own. For example, it might group similar items together or identify trends, like clustering customers based on their buying habits.
